### ISSUE TITLE
buildpack: can't build in tsuru-bootstrap

### ISSUE THREAD

**Issue by guestisp** [tsuru/platforms#6]
I'm trying to add `buildpack` platform in `tsuru-bootstrap` environment:
Download output was truncated for posting here on GH, that's why progress bar aren't shown properly

```
$ tsuru-admin platform-add buildpack -d https://github.com/tsuru/platforms/raw/master/buildpack/Dockerfile
Step 1 : FROM progrium/buildstep
Pulling from progrium/buildstep
Pulling fs layer
Pulling fs layer
Pulling fs layer
Pulling fs layer
Pulling fs layer
Pulling fs layer
Pulling fs layer
Pulling fs layer
Downloading [==================================================>]     32 B/32 B
Extracting [==================================================>]    337 B/337 B
Extracting [==================================================>]    337 B/337 B
Extracting [==================================================>]    337 B/337 B
Pull complete
Digest: sha256:70ecdcb8660431593f56b0615da75ef013a0fd58d24f388341e731840c87ce81
Status: Downloaded newer image for progrium/buildstep:latest
 ---> d6d2e6f5fe3c
Step 2 : RUN locale-gen en_US.UTF-8
 ---> Running in 9a7123f056a2
Generating locales...
  en_US.UTF-8... done
Generation complete.
 ---> 94ac1884869b
Removing intermediate container 9a7123f056a2
Step 3 : ENV LANG en_US.UTF-8 LANGUAGE en_US:en LC_ALL en_US.UTF-8 DEBIAN_FRONTEND noninteractive
 ---> Running in 5dc8dde08984
 ---> 7bba448a83ea
Removing intermediate container 5dc8dde08984
Step 4 : RUN apt-get update
 ---> Running in 5cc6c902bfeb
Ign http://archive.ubuntu.com trusty InRelease
Get:1 http://archive.ubuntu.com trusty-security InRelease [64.4 kB]
Get:2 http://archive.ubuntu.com trusty-updates InRelease [64.4 kB]
Hit http://archive.ubuntu.com trusty Release.gpg
Get:3 http://archive.ubuntu.com trusty-security/main amd64 Packages [476 kB]
Get:4 http://archive.ubuntu.com trusty-updates/main amd64 Packages [835 kB]
Hit http://archive.ubuntu.com trusty Release
Hit http://archive.ubuntu.com trusty/main amd64 Packages
Hit http://archive.ubuntu.com trusty/universe amd64 Packages
Fetched 1,440 kB in 6s (216 kB/s)
Reading package lists...
 ---> cd3ddedf4d8a
Removing intermediate container 5cc6c902bfeb
Step 5 : RUN apt-get install curl sudo -y --force-yes
 ---> Running in 468e853e5144
Reading package lists...
Building dependency tree...
Reading state information...
The following extra packages will be installed:
  libcurl3 libcurl4-openssl-dev
Suggested packages:
  libcurl4-doc libcurl3-dbg
The following NEW packages will be installed:
  sudo
The following packages will be upgraded:
  curl libcurl3 libcurl4-openssl-dev
3 upgraded, 1 newly installed, 0 to remove and 180 not upgraded.
Need to get 883 kB of archives.
After this operation, 1,544 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu/ trusty-security/main curl amd64 7.35.0-1ubuntu2.5 [123 kB]
Get:2 http://archive.ubuntu.com/ubuntu/ trusty-security/main libcurl4-openssl-dev amd64 7.35.0-1ubuntu2.5 [245 kB]
Get:3 http://archive.ubuntu.com/ubuntu/ trusty-security/main libcurl3 amd64 7.35.0-1ubuntu2.5 [173 kB]
Get:4 http://archive.ubuntu.com/ubuntu/ trusty-updates/main sudo amd64 1.8.9p5-1ubuntu1.2 [342 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 883 kB in 1s (791 kB/s)
(Reading database ... 48111 files and directories currently installed.)
Preparing to unpack .../curl_7.35.0-1ubuntu2.5_amd64.deb ...
Unpacking curl (7.35.0-1ubuntu2.5) over (7.35.0-1ubuntu2.3) ...
Preparing to unpack .../libcurl4-openssl-dev_7.35.0-1ubuntu2.5_amd64.deb ...
Unpacking libcurl4-openssl-dev:amd64 (7.35.0-1ubuntu2.5) over (7.35.0-1ubuntu2.3) ...
Preparing to unpack .../libcurl3_7.35.0-1ubuntu2.5_amd64.deb ...
Unpacking libcurl3:amd64 (7.35.0-1ubuntu2.5) over (7.35.0-1ubuntu2.3) ...
Selecting previously unselected package sudo.
Preparing to unpack .../sudo_1.8.9p5-1ubuntu1.2_amd64.deb ...
Unpacking sudo (1.8.9p5-1ubuntu1.2) ...
Setting up libcurl3:amd64 (7.35.0-1ubuntu2.5) ...
Setting up curl (7.35.0-1ubuntu2.5) ...
Setting up libcurl4-openssl-dev:amd64 (7.35.0-1ubuntu2.5) ...
Setting up sudo (1.8.9p5-1ubuntu1.2) ...
Processing triggers for libc-bin (2.19-0ubuntu6.6) ...
 ---> 4fd754d7244e
Removing intermediate container 468e853e5144
Step 6 : RUN curl -sLo base-platform.tar.gz https://github.com/tsuru/base-platform/tarball/master
 ---> Running in fe49da41a57c
 ---> 9870d5a66f9b
Removing intermediate container fe49da41a57c
Step 7 : RUN mkdir -p /var/lib/tsuru/base
 ---> Running in e5445190da4c
 ---> 0348788a2693
Removing intermediate container e5445190da4c
Step 8 : RUN tar -xvf base-platform.tar.gz -C /var/lib/tsuru/base --strip 1
 ---> Running in 191b65822bcc
tsuru-base-platform-67e836c/AUTHORS
tsuru-base-platform-67e836c/CONTRIBUTORS
tsuru-base-platform-67e836c/Dockerfile
tsuru-base-platform-67e836c/LICENSE
tsuru-base-platform-67e836c/README.md
tsuru-base-platform-67e836c/deploy
tsuru-base-platform-67e836c/install
tsuru-base-platform-67e836c/rc/
tsuru-base-platform-67e836c/rc/config
tsuru-base-platform-67e836c/rc/os_dependencies
 ---> 97e0dead8baf
Removing intermediate container 191b65822bcc
Step 9 : RUN rm base-platform.tar.gz
 ---> Running in 331538de3a07
 ---> f1462f5c7782
Removing intermediate container 331538de3a07
Step 10 : RUN mkdir -p /var/lib/tsuru/buildpack
 ---> Running in 56222f608eb9
 ---> 45f7c905f1c5
Removing intermediate container 56222f608eb9
Step 11 : ADD . /var/lib/tsuru/buildpack
 ---> 3cff47dc7344
Removing intermediate container 38e2744e0c9e
Step 12 : RUN cp /var/lib/tsuru/buildpack/deploy /var/lib/tsuru
 ---> Running in aca2ccdf3569
cp: cannot stat ‘/var/lib/tsuru/buildpack/deploy’: No such file or directory
error in docker node "http://127.0.0.1:2375": The command '/bin/sh -c cp /var/lib/tsuru/buildpack/deploy /var/lib/tsuru' returned a non-zero code: 1
Error: Failed to add new platform.
```

**Comment by guestisp** [comment]
The same happens when trying to use `php` platform:

```
$ tsuru-admin platform-add php -d https://raw.githubusercontent.com/tsuru/platforms/master/php/Dockerfile
Step 1 : FROM tsuru/base-platform
Pulling from tsuru/base-platform
Digest: sha256:922d4c495938d4de04ff91e9180b63faa40b30a56ade21c6f2bcef235004a28f
Status: Downloaded newer image for tsuru/base-platform:latest
 ---> b0c3d41bee57
Step 2 : RUN echo 'deb http://archive.ubuntu.com/ubuntu/ vivid main restricted universe multiverse' >> /etc/apt/sources.list
 ---> Running in b4fb6f30e2ec
 ---> 75d07f544e03
Removing intermediate container b4fb6f30e2ec
Step 3 : RUN echo 'deb http://archive.ubuntu.com/ubuntu/ vivid-security main restricted universe multiverse' >> /etc/apt/sources.list
 ---> Running in 6840955b568a
 ---> 4106abdf909e
Removing intermediate container 6840955b568a
Step 4 : RUN echo 'APT::Default-Release "trusty";' >> /etc/apt/apt.conf.d/01ubuntu
 ---> Running in 406f36bdc722
 ---> d6d8ff1c7a9b
Removing intermediate container 406f36bdc722
Step 5 : ADD . /var/lib/tsuru/php
 ---> d5884312f9bb
Removing intermediate container 5340d2c86bf6
Step 6 : RUN cp /var/lib/tsuru/php/deploy /var/lib/tsuru
 ---> Running in 1d888cc0401b
cp: cannot stat ‘/var/lib/tsuru/php/deploy’: No such file or directory
error in docker node "http://127.0.0.1:2375": The command '/bin/sh -c cp /var/lib/tsuru/php/deploy /var/lib/tsuru' returned a non-zero code: 1
Error: Failed to add new platform.
```

**Comment by fsouza** [comment]
@guestisp platforms in this repository are automatically built by Docker Hub. In order to use then prebuilt, you need to either have tsurud 0.13+ and just run ``tsuru-admin platform-add [platform-name]`` or use the Dockerfiles in the basebuilder project (you can see that the [Dockerfile source](https://raw.githubusercontent.com/tsuru/basebuilder/master/buildpack/Dockerfile) is very simple).

There are instructions here: https://github.com/tsuru/platforms#installing-platforms.

In your case, the command should be:

```
% tsuru-admin platform-add buildpack -d https://raw.githubusercontent.com/tsuru/basebuilder/master/buildpack/Dockerfile
```

Some background is available in tsuru/basebuilder#98.

**Comment by guestisp** [comment]
Solved with your suggestion
